lib/utils/fs_test.go

```diff
@@ -90,3 +92,54 @@ func TestLocks(t *testing.T) {
 
 	require.NoError(t, unlock())
 }
+
+func TestOverwriteFile(t *testing.T) {
+	have := []byte("Sensitive Information")
+
+	tmpFile, err := os.CreateTemp("", "teleport-overwrite-file-test-")
+	if err != nil {
+		t.Fatalf("Unable to create tmp file: %s\n", err)
+	}
+	t.Cleanup(func() {
+		if err := os.Remove(tmpFile.Name()); err != nil {
+			t.Fatalf("Error calling os.Remove: %s\n", err)
+		}
+	})
+
+	if _, err := tmpFile.Write(have); err != nil {
+		tmpFile.Close()
+		t.Fatalf("Error writing to tmp file: %s\n", err)
+	}
+	if err := tmpFile.Close(); err != nil {
+		t.Fatalf("Unable to close tmp file: %s\n", err)
```

Comment:
We tend to make use of `require`/`assert` testify packages for error assertions, I find it come out a little less verbose than this when handling a lot of errors.

Reply:
Done.